package.json

```diff
@@ -1,8 +1,15 @@
 {
-  "name": "cordova-plugin-iroot",
-  "version": "0.8.0",
+  "name": "@aerogear/cordova-plugin-iroot",
+  "version": "0.8.1",
```

Comment:
In actual release - let's mention what we actually fixed upstream (compaiblity wqith latest cordova).

Reply:
do you mean create changelog file or just write the changelog in the GitHub release page?